vpd-manager/src/manager.cpp

```diff
@@ -29,7 +29,18 @@ Manager::Manager(
     try
     {
 #ifdef IBM_SYSTEM
-        m_worker = std::make_shared<Worker>(INVENTORY_JSON_DEFAULT);
+        if (dbusUtility::isChassisPowerOn())
```

Comment:
what happens when vpd-manager starts when chassis state is transitioning to On

Reply:
Then we consider it as chassis power off and go ahead.

Reply:
We restrict the collection of specific FRUs at power on currently. For rest of the states, we go ahead with the general collection.